docs/08-troubleshooting.md

```diff
@@ -114,6 +114,8 @@ node --max-old-space-size=8192 node_modules/rollup/bin/rollup -c
 
 increasing `--max-old-space-size` as needed. Note that this number can safely surpass your available physical memory. In that case, Node will start paging memory to disk as needed.
 
+You may reduce memory pressure by introducing code splitting by using dynamic imports, importing just specific modules rather than whole dependencies, disabling sourcemaps, or increasing the size of your swap space.
```

Comment:
Does code-splitting really help? I definitely fully agree with everything else and I would also leave it in here, but it intrigues me as I would not expect it to make a lot of difference for Rollup at the moment.